### PR TITLE
commiting work

### PR DESCRIPTION
**Pull request by someone20dz**
here is my work missing timer

### CODE REVIEW

**Comment by someone20dz** [comment]
- [ ] Adding typography for ChanR icon (@DaLiteralPanda 's typography choice from first version)
- [ ] Adding motto for ChanR 
- [ ] Updating Background Image
- [ ] Adding countdown

File: assets/js/timer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  document.getElementById("minutes").innerHTML = minutes
+  document.getElementById("seconds").innerHTML = seconds
+  // If the count down is finished, write some text
+  if (distance < 0) {
+    clearInterval(x);
+    document.getElementById("timer").innerHTML = "EXPIRED";
+  }
+}, 1000);
+
+// END Count Down
```
Comment (TheChanRProject) [inline assets/js/timer.js:33]: Super cool

File: chanr_hs_immersive.html [hunk 2]
```diff
@@ -138,6 +138,28 @@ <h2 class="title">
...
+                          <td id="minutes"></td>
+                          <td id="seconds"></td>
+                      </tr>
+                    </table>
+                  </div>
+                </div>
+
               </div>
             </div>
           </div>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:165]: Awesome way to merge yours and Rishit's work together

File: chanr_hs_immersive.html [hunk 3]
```diff
@@ -2976,6 +2998,7 @@ <h5 class="info-title">Find us at the office</h5>
     </div>
   </footer>
   <!--   Core JS Files   -->
+  <script src="./assets/js/timer.js" type="text/javascript"></script>
   <script src="./assets/js/core/jquery.min.js" type="text/javascript"></script>
   <script src="./assets/js/core/popper.min.js" type="text/javascript"></script>
   <script src="./assets/js/core/bootstrap.min.js" type="text/javascript"></script>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:3004]: Yep exactly. Nice import.

**Review by TheChanRProject — APPROVED** [review]
Looks great 👍  

Approved.

**Pull request merged**